docs/source/usecases/homesale-forecast.rst

```diff
@@ -0,0 +1,135 @@
+
+  CREATE FUNCTION IF NOT EXISTS HomeSaleForecast FROM
+    (
+      SELECT propertytype, datesold, price
+      FROM postgres_data.home_sales
+      WHERE bedrooms = 3 AND postcode = 2607
+    )
+  TYPE Forecasting
+  PREDICT 'price'
+  TIME 'datesold'
+  ID 'propertytype'
+  FREQUENCY 'W';
+
+While running this query, EvaDB will train different models respectively for different property types (i.e., house, unit). Here is the query's output ``DataFrame``:
+
+.. note::
+
+   Go over :ref:`forecast` page on fine tuning the forecast model.
+
+.. code-block:: 
+
+   +----------------------------------------------+
+   | Function HomeSaleForecast successfully added |
+   +----------------------------------------------+
+
+Predict the Home Price using the Trained Model
+----------------------------------------------
+
+Next we use the trained ``HomeSaleForecast`` to predict the home sale price for next 3 weeks.
+
+.. code-block:: sql
+
+   SELECT HomeSaleForecast(3);
+
+The input of the trained model is the horizon, the steps we want to forecast in the future. Here is the query's output ``DataFrame``:
+
+.. code-block::
+
+   +-------------------------------+---------------------------+------------------------+
+   | homesaleforecast.propertytype | homesaleforecast.datesold | homesaleforecast.price |
+   +-------------------------------+---------------------------+------------------------+
+   |                         house |                2019-07-21 |                 766572 |
+   |                         house |                2019-07-28 |                 766572 |
+   |                         house |                2019-08-04 |                 766572 |
+   |                          unit |                2018-12-23 |                 417229 |
+   |                          unit |                2018-12-30 |                 409601 |
+   |                          unit |                2019-01-06 |                 402112 |
+   +-------------------------------+---------------------------+------------------------+
+
+In the previous ``CREATE FUNCTION`` query, we specified the ``propertytype`` as the ID column because the price gap between houses and units can be large. Accordingly, EvaDB will train independent time-series models for house and unit to capture the difference as shown in the above output. 
+
+We can further use ``ORDER BY`` to find out which month in the following year has the lower price.
+
+.. code-block:: sql
+
+   SELECT * 
+   FROM (SELECT HomeSaleForecast(12)) AS HomeSale
+   ORDER BY price
+   LIMIT 1;
+
+Here is the query's output:
+
+.. code-block::
+
+   +-----------------------+-------------------+----------------+
+   | HomeSale.propertytype | HomeSale.datesold | HomeSale.price |
+   +-----------------------+-------------------+----------------+
+   |                  unit |        2019-03-10 |         340584 |
+   +-----------------------|-------------------|----------------|
+   
+
+.. include:: ../shared/nlp.rst
```

Comment:
We can remove the `nlp` footer. Maybe add a `forecast` footer in the future as we add more usecases.